setup.py

```diff
@@ -43,30 +50,40 @@ def read_file(fname):
     data_files=[("", ["LICENSE"])],
     package_data={
         "tk_toolchain": [
+            os.path.join("tk_testengine", "*"),
             os.path.join("cmd_line_tools", "tk_docs_preview", "sphinx_data", "*"),
+            os.path.join("cmd_line_tools", "tk_run_app", "config", "env", "*"),
+            os.path.join("cmd_line_tools", "tk_run_app", "config", "core", "*"),
+            os.path.join(
+                "cmd_line_tools", "tk_run_app", "config", "core", "hooks", "*"
+            ),
             os.path.join(
                 "cmd_line_tools", "tk_docs_preview", "sphinx_data", "_static", "*"
             ),
-        ],
-        "pytest_tank_test": [os.path.join("tk-testengine", "*")],
+        ]
     },
-    python_requires=">=2.7.0",
+    python_requires=">=2.6.0",
     install_requires=[
         # Tests
-        "pytest==4.6.6",
-        "pytest-cov==2.6.1",
-        "mock",
+        "pytest==4.6.6" if is_python_27_or_greater else "pytest<3.3",
```

Comment:
As you'll see, people have been dropping support for 2.6 over time, so not everything we use can be used with the latest version.